messages.proto

```diff
@@ -30,32 +30,64 @@ enum MessageRight {
   CAN_DELETE_MESSAGE = 2;
 }
 
+// Full message with all fields
 message Message {
   ChatPreview chat = 8;
   string id = 1;
   uint64 created_at = 3;
-  uint64 updated_at = 9;
+  uint64 changed_at = 9; // when equal to created_at, message was not edited
```

Comment:
Why did you decided to rename this field?

Reply:
* `updated_at` reserved for any update (e.g. reactions count)
* `changed_at` means that message edited by author, it happens only on text / uploads / reply / forwards change